app/resources/configuration_resource.rb

```diff
@@ -0,0 +1,18 @@
+require 'okapi'
+
+class ConfigurationResource < JSONAPI::Resource
+  attributes :customer_id, :api_key
+
+  def self.find_by_key(key, options = {})
+    context = options[:context]
+    okapi = context[:okapi]
+    model = Configuration.new(okapi)
+    model.load!
+
+    resource_for_model(model).new(model, context)
+  end
+
+  def model_error_messages
```

Comment:
We should document what this is, and how it fits into the jsonapi resource request cycle.